src/account/interface.cairo

```diff
@@ -25,31 +25,33 @@ trait IDeclarer<TState> {
     fn __validate_declare__(self: @TState, class_hash: felt252) -> felt252;
 }
 
+//
+// Account ABI
+//
+
 #[starknet::interface]
 trait AccountABI<TState> {
+    // ISRC6
     fn __execute__(self: @TState, calls: Array<Call>) -> Array<Span<felt252>>;
     fn __validate__(self: @TState, calls: Array<Call>) -> felt252;
+    fn is_valid_signature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
+
+    // ISRC5
+    fn supports_interface(self: @TState, interface_id: felt252) -> bool;
+
+    // IDeclarer
```

Comment:
is `IDeclarer` defined anywhere? if not let's put it with `__validate_deploy__` as non-standard, I think it can be confusing otherwise.

Reply:
It's defined just above on [line 24](https://github.com/OpenZeppelin/cairo-contracts/pull/741/files#diff-adb12f74c6c498f1c4652a9d393437bacfb0504545337793ea9fe3f0f00ef59dR24). I see your point though since it is non-standard. I'll move it to `Non-standard` to keep this simple

Reply:
> I'll move it to Non-standard to keep this simple

Actually, I take it back. I think there's value in identifying non-standard extensions by their interface when applicable. I vote we keep it